src/AppBundle/Service/OrderManager.php

```diff
@@ -194,6 +187,39 @@ public function capturePayment(OrderInterface $order)
             $stripePayment->setLastError($e->getMessage());
             $stateMachine->apply(PaymentTransitions::TRANSITION_FAIL);
         }
+
+        $stripeTransfer = new StripePayment();
```

Comment:
Ok.

"This way, we make sure the transfer is created only when the charge is captured successfully, and we may be able to replay transfers." >> This actually the case because we are using this [Stripe feature](https://stripe.com/docs/connect/charges-transfers#transfer-availability).

Reply:
Can I delete the `processing` payment state? I don't think it is used and it cleans a little bit the mess